src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String DEFAULT_TITLE = "fruit,quantity" + System.lineSeparator();
```

Comment:
you can remove System.lineSeparator from DEFAULT_TITLE and add it as first operation in which for-each iteration